src/main/java/Problem2.java

```diff
@@ -3,5 +3,36 @@
     // Do not change signature (function name, parameters)
     public static void bubbleSort(int[] data) {
         // Add your solution
+        //This is taken from the website you linked, and while I could redo the code,
+        //I felt that there was no real point. I will however explain the logic so that
```

Comment:
I disagree with this. even if we can get it from some website, there's always some benefit by redoing it ourselves.